app/src/main/java/com/c23ps266/capstoneprojectnew/ui/LoginActivity.kt

```diff
@@ -1,82 +1,56 @@
 package com.c23ps266.capstoneprojectnew.ui
 
 import android.content.Intent
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
-import com.c23ps266.capstoneprojectnew.R
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.ViewModelProvider
 import com.c23ps266.capstoneprojectnew.databinding.ActivityLoginBinding
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.auth.FirebaseUser
 
 class LoginActivity : AppCompatActivity() {
     private lateinit var binding: ActivityLoginBinding
-//    private lateinit var googleSignInClient: GoogleSignInClient
-//    private lateinit var auth: FirebaseAuth
+    private lateinit var viewModel: LoginViewModel
+    private lateinit var firebaseAuth: FirebaseAuth
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = ActivityLoginBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
+        firebaseAuth = FirebaseAuth.getInstance()
```

Comment:
You don't have to call this firebase auth from here, it has been provided by the viewmode